src/cps.cpp

```diff
@@ -5,6 +5,17 @@
 #include "cps.hpp"
 #include <cmath>
 
+// --- Shape to PostScript File ---
+
```

Comment:
If we make The create PostScript function take an ostream, it would handle this for us eliminating the need for a special function.